tests/core/full_node/test_mempool.py

```diff
@@ -1449,15 +1465,33 @@ async def test_stealing_fee(self, two_nodes):
         assert status == MempoolInclusionStatus.FAILED
 
     @pytest.mark.asyncio
-    async def test_double_spend_same_bundle(self, two_nodes):
+    async def test_double_spend_same_bundle(self, bt, two_nodes, wallet_a):
```

Comment:
This test looks substantially different than the prior implementation. Just curious what led to the changes.

Reply:
I'll take a look. Mindnumbing number of changes, plus conflicts with an earlier rebase, thanks.

Reply:
Thank you. This looks like a rebase snafu. `test_agg_sig_condition` as well. Looking into it